cypress/page/shoping-cart.page.ts

```diff
@@ -0,0 +1,21 @@
+class ShopingCartPage {
+    private ShopingCart: string;
+    private CartNavigation: string
+
+    constructor() {
+        this.ShopingCart = '[style*="display: block;"] .button-container > a';
+        this.CartNavigation = '.cart_navigation span'   
```

Comment:
This is referring to the proceed to checkout button, let's name this property in that way.